### PR TITLE
Update Bescort.lic

### PR DESCRIPTION
**Pull request by BinuDR**
Use equipment manager to change into swimming gearset before swimming the Segoltha, then change back to standard when done.  My swimming gearset is my standard set minus my torso armor.  It makes swimming much easier.

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -716,6 +716,8 @@ class Bescort
   end
 
   def segoltha(mode)
+    EquipmentManager.new.empty_hands
+    EquipmentManager.new.wear_equipment_set?('swimming')
```
Comment (rpherbig) [inline bescort.lic]: What does this do if there is no `swimming` set defined? I assume it either would remove all gear or change nothing?

Comment (BinuDR) [reply bescort.lic]: Yes.

Comment (rpherbig) [reply bescort.lic]: Sorry, I'm not sure I understand. Which thing does it do if that set is not defined?

Comment (BinuDR) [reply bescort.lic]: The Yes was in response to your original inquiry of if the 3rd addition was supposed to be:
'EquipmentManager.new.wear_equipment_set?('standard')'

If there is no 'swimming' gearset defined, it does this:
`echo("Could not find gear set #{set_name}")`

**Pull request merged**